sdk/metric/doc.go

```diff
@@ -12,36 +12,107 @@
+make metric measurements. Also, it is an object that should be used to
+control the life-cycle (start, flush, and shutdown) of the SDK.
+
+A MeterProvider needs to be configured to export the measured data, this is
+done by configuring it with a Reader implementation (using the WithReader
+MeterProviderOption). Readers take two forms: ones that push to an endpoint
+(NewPeriodicReader), and ones that an endpoint pulls from. See
+[go.opentelemetry.io/otel/exporters] for exporters that can be used as
+or with these Readers.
+
+Each Reader, when registered with the MeterProvider, can be augmented with a
+View. Views allow users that run OpenTelemetry instrumented code to modify
+the generated data of that instrumentation.
+
+The data generated by a MeterProvider needs to include information about its
+origin. A MeterProvider needs to be configured with a Resource, using the
+WithResource MeterProviderOption, to include this information. This Resource
+should be used to describe the unique runtime environment instrumented code
+is being run on. That way when multiple instances of the code are collected
+at a single endpoint their origin is decipherable.
+
+See [go.opentelemetry.io/otel/metric] for more information about
+the metric API.
+
+# Experimental
+
+The metric SDK contains features that have not yet stabilized in the
+OpenTelemetry specification. These features are added to the OpenTelemetry Go
+metric SDK prior to stabilization in the specification so that users can start
+experimenting with them and provide feedback.
+
+These feature may change in backwards incompatible ways as feedback is applied.
+
+Experimental features do not fall within the scope of the OpenTelemetry Go
+versioning and stability policy. These features may be removed or modified in
+successive version releases, including patch versions.
+
+When an experimental feature is promoted to a stable feature, a migration path
+will be included in the changelog entry of the release. There is no guarantee
+that any environment variable feature flags that enabled the experimental
+feature will be supported by the stable version. If they are supported, they
+may be accompanied with a deprecation notice stating a timeline for the removal
+of that support.
+
+# Cardinality Limit (experimental)
+
+The cardinality limit is the hard limit on the number of metric streams that
+can be collected for a single instrument.
+
+This experimental feature can be enabled by setting the
+OTEL_GO_X_CARDINALITY_LIMIT environment value. The value must be an integer
+value. All other values are ignored. If the value set is less than or equal
+to 0, no limit will be applied.
+
+This is how you can set the cardinality limit to 2000:
+
+	export OTEL_GO_X_CARDINALITY_LIMIT=2000
+
+This is how you can set an infinite cardinality limit
+(functionally equivalent to disabling the feature):
+
+	export OTEL_GO_X_CARDINALITY_LIMIT=-1
+
+This is how you can disable the cardinality limit.
```

Comment:
This reads very repetitive now that it is not in its own section.

Reply:
Do you have any proposal to make it better? I do not find repetition is doing any harm here.

Reply:
I would keep it the way it already is in its own document within its own section, and link to the document from the Go docs.